File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/INTRODUCCION/Ejercicio5Py_java/Persona.py
class Persona:
    def __init__(self, nombre, paterno, materno, edad, ci):
        self.nombre = nombre
        self.paterno = paterno
        self.materno = materno
        self.edad = edad
        self.ci = ci

    def mostrarDatos(self):
        print(f"Nombre completo: {self.nombre} {self.paterno} {self.materno}")
        print(f"Edad: {self.edad}")
        print(f"CI: {self.ci}")

    def mayorDeEdad(self):
        if self.edad >= 18:
            print(f"{self.nombre} es mayor de edad.")
            return True
        else:
            print(f"{self.nombre} NO es mayor de edad.")
            return False

    def modificarEdad(self, nuevo):
        if nuevo >= 0:
            print(f"Edad modificada de {self.edad} a {nuevo}.")
            self.edad = nuevo
        else:
            print("Edad inválida. No se realizó el cambio.")

    def getApellidoPaterno(self):
        return self.paterno

File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/INTRODUCCION/Ejercicio5Py_java/Persona.java
package Ejercicio5Py_java;

public class Persona {
    private String nombre;
    private String paterno;
    private String materno;
    private int edad;
    private String ci;

    public Persona(String nombre, String paterno, String materno, int edad, String ci) {
        this.nombre = nombre;
        this.paterno = paterno;
        this.materno = materno;
        this.edad = edad;
        this.ci = ci;
    }

    public void mostrarDatos() {
        System.out.println("Nombre completo: " + nombre + " " + paterno + " " + materno);
        System.out.println("Edad: " + edad);
        System.out.println("CI: " + ci);
    }

    public boolean mayorDeEdad() {
        return edad >= 18;
    }

    public void modificarEdad(int nuevaEdad) {
        if (nuevaEdad > 0) {
            this.edad = nuevaEdad;
        } else {
            System.out.println("Edad inválida.");
        }
    }

    public String getPaterno() {
        return this.paterno;
    }
}
